.github/workflows/gh-pages.yml

```diff
@@ -6,26 +6,40 @@ on:
 
 jobs:
 
-  documentation-job:
+  build-documentation:
     runs-on: ubuntu-24.04
-
+    permissions:
+      contents: read
     steps:
       - name: SCM Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@1.0.1
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.3.0
 
       - name: Build Documentation
         run: |
           poetry run -- nox -s docs:multiversion
+          rm -r .html-documentation/*/.doctrees
 
-      - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@v4.7.2
+      - name: Upload artifact
```

Comment:
@tkilias , @tomuben With this, we'd want to update the settings. It seems like there is some minimal documentation being created, but it isn't listed in the same way as the other pages. It's linked in the README but it doesn't have a counterpart like

![image](https://github.com/user-attachments/assets/a04ae635-67f3-420b-b383-b05bba79142b)

Is that intentional?